daemon/openrazer_daemon/dbus_services/dbus_methods/mambaelite.py

```diff
@@ -0,0 +1,188 @@
+from openrazer_daemon.dbus_services import endpoint
+from openrazer_daemon.dbus_services.dbus_methods.lanceheadte import get_left_brightness as _get_left_brightness, \
+    get_right_brightness as _get_right_brightness, \
+    set_left_wave as _set_left_wave, set_right_wave as _set_right_wave, \
+    set_left_static as _set_left_static, set_right_static as _set_right_static, \
+    set_left_spectrum as _set_left_spectrum, set_right_spectrum as _set_right_spectrum, \
+    set_left_none as _set_left_none, set_right_none as _set_right_none, \
+    set_left_reactive as _set_left_reactive, set_right_reactive as _set_right_reactive, \
+    set_left_breath_random as _set_left_breath_random, set_right_breath_random as _set_right_breath_random, \
+    set_left_breath_single as _set_left_breath_single, set_right_breath_single as _set_right_breath_single, \
+    set_left_breath_dual as _set_left_breath_dual, set_right_breath_dual as _set_right_breath_dual
+
+
+@endpoint('razer.device.lighting.brightness', 'getBrightness', out_sig='d')
+def get_brightness_mamba_elite(self):
+    """
+    Get the device's brightness not supported and splited into set of brightnesses(scroll, logo left and right)
+    Current brightness == a left+right
+
+    :return: Brightness
+    :rtype: float
+    """
+    self.logger.debug("DBus call get_brightness")
+    return (_get_left_brightness(self) + _get_right_brightness(self)) / 2
```

Comment:
Do we need those "fake" methods? We have scroll, logo, left side & right side zones, so why add a zone being both left&right?

Reply:
Method "get_brightness" complements "set_brightness" (set brightness level for all components), Another devices has both methods, so implementing this one does simplifiy control application.

Reply:
In my opinion the driver/daemon shouldn't make opaque wrappers that frontends have no idea what they are doing. If you want to "synchronize" all lighting zones on the device in the frontend, it should be done in the frontend and not via some wrapper behind that.

Reply:
Absolutely agree! You must not smoking on gas station - so this code is not placed in kernel module, because it is a task of high level wrapper, which are used for simplify access through d-bus and decrease device-specific stuff.

Reply:
That's not what I have meant and I'm pretty sure you know that...

In short: I'm the maintainer, if you want to get the pull request merged please accept my position as maintainer and implement the requested changes.

Reply:
Or probably we just need to do a fork instead of all this silly nitpicking —.—

Reply:
Dear **@Smarre**, There is no problem to fork (and I have already fork the repository for  Mamba Elite development) - but maintainers support another devices so prefer to merge into master brunch, and for this task I ready to pull down code development requirements in *optional cases* (like [Linux kernel coding style](https://01.org/linuxgraphics/gfx-docs/drm/process/coding-style.html) - the specifications has been wrote by more experience and smart people) if maintainer wants - it is not good, but acceptable and users would get solution.
 
Dear **The Maintainer @z3ntu**, I would thank you for working and coordinate the openrazer project. But as you notice  (and for my luck) - You are *just* maintainer and I may set aside to do controversial solutions - the library's (intermediate layer's) target is to simplify and unify.